saleor/graphql/core/validators.py

```diff
@@ -54,6 +54,19 @@ def validate_price_precision(value: Optional["Decimal"], currency: str):
         )
 
 
+def validate_price_max_value(value: Optional["Decimal"], max_value=10 ** 9):
```

Comment:
I would rename function to decimal instead of price. It could be use for both cases as you're operating here on Decimal which could be related to something else than price

Reply:
Why value is Optional?

Reply:
@korycins function renamed
@L3str4nge as we discussed - simplified function and dropped unnecessary checks